hack/util.sh

```diff
@@ -570,10 +570,11 @@ function cleanup_openshift {
 		set -u
 	fi
 
-	if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then 
+	# TODO soltysh: restore the if back once #8399 is resolved
+	# if grep -q 'no Docker socket found' "${LOG_DIR}/openshift.log"; then
 		# the Docker daemon crashed, we need the logs
-		journalctl --unit docker.service --since -4hours > "${LOG_DIR}/docker.log"
-	fi
+	journalctl --unit docker.service --since -4hours > "${LOG_DIR}/docker.log"
```

Comment:
Slim this down to 15 minutes, that may still be too much logs but we only need the last run.